docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -44,214 +45,504 @@ but that effect is (completely) mediated by another risk factor R2.
+  as described in much more detail on :ref:`the page about confounding <confounding>`.
+  The reason experimental evidence is the gold standard is that it adjusts for *all* confounders,
+  known and unknown.
+* If the association is adjusted *only* for confounders, it is simply a measure of the total effect.
+* However, if the association is adjusted for some mediators in addition to all (known) confounders,
+  each of the mediators it is adjusted for "blocks" the pathway(s) it is on.
+  The association then reflects the combination (as described in the previous section) of all
+  the *remaining/unblocked* pathways.
+  If at least one mediator on every mediated pathway is adjusted for, all that is left is the direct
+  pathway, and the association can be considered the direct effect.
+
+It is key to understand that the definition of a "direct effect" depends on what mediators
+we are modeling.
+The direct effect arrow in our causal diagram should be informed by an association measure adjusted
+for the mediators that we model, but *not* adjusted for mediators we do not model.
+
+Combining this understanding with the simple cases of combining and composing log-linear and linear effects discussed
+in the previous sections, we can say the following about an effect adjusted for the set of
+mediated pathways :math:`M`:
 
-Note that while the ideal would be evidence from a randomized controlled trial,
-therefore controlling for *all* confounders (known and unknown), that frequently isn't possible,
-including in this example:
-there isn't a way to randomly assign people BMIs or SBPs since these variables
-are not directly manipulable.
+.. math::
+
+  RR_\text{r→o,adjusted for M} = \frac{RR_\text{r→o,total}}{\prod_{m \in M}{RR_\text{r→o, via m}}}
 
-GBD-modeled mediation
-+++++++++++++++++++++
+in the log-linear case, and
+
+.. math::
+
+  \Delta_\text{r→o,adjusted for M} = \Delta_\text{r→o,total} - \sum_{m \in M}{\Delta_\text{r→o, via m}}
+
+in the linear case.
+
+Composing pathway effects from individual causal effects
+--------------------------------------------------------
+
+While thinking in terms of causal pathways is helpful in analyzing how to split
+or combine multiple pathways, the effect of an entire pathway
+never appears in our simulation.
+It is each individual link in the chain -- the causal effect between just two factors
+at a time -- that we model.
+Frequently, we will have data about the individual effects in the chain, and will want to calculate
+the effect of the whole pathway.
+We call this "composing" the effects, because it turns out that the way this works is
+mathematical function composition.
+(More rarely, we may start from a pathway effect and decompose it, which is the same
+operation, just in reverse.)
+
+We will discuss two common and easy cases here, and leave the more general principles
+to be fleshed out in the future.
+
+Case 1
+++++++
+
+.. image:: composition-case-1.drawio.svg
+
```

Comment:
I don't have a great solution, and the text is clear, but these diagrams look to me like you just keep the x effect and ignore the + effect. 

Maybe we could put x^+

Reply:
Yeah, that's a good point.

The intention here is to diagram the effect _type_ (linear or log-linear) but not the effect _size_. I think the notation gets a little unwieldy if we try to include the effect size -- for example, in your suggestion, if I put both an `x` and a `+` on the same arrow, it might not be clear anymore what type of effect it is (`x`). Doubly so in the next diagram, where I would need to use a times symbol to show how to calculate the _size_ of a `+` effect.

Definitely open to ideas about either how to make the effect _type_ notation clearer, or more ideas for an unambiguous effect _size_ notation!

Reply:
Hmm okay, I see your point. Could we have a different symbol for "to the power of"? So then three total symbols: +, x, and maybe e or pow? 

+ and + turn to x. x and + turn to pow. So on? 

Otherwise we can leave as is and not overcomplicate it. As I said - the text is very clear once you read it and see what's happening.

Reply:
I think there are two separate attributes for each arrow in the diagram: what _type_ of effect is it, and how you _calculate_ the effect size (RR for a log-linear `x` effect, delta for a linear `+` effect).

The composition rules here describe both the resulting effect type and its size:
* A `+` effect _type_ with an effect size of $a$ and a `x` effect type with an effect size of $b$ compose into a `x` effect with effect size $b^a$.
* A `+` effect type with an effect size of $a$ and a `+` effect type with an effect size of $b$ compose into a `+` effect type with an effect size of $a \times b$.

Maybe I am misunderstanding what you mean, but I don't think there are three types (+, x, pow) -- there are only two. It is just that the power operation is used in the calculation of the effect _size_ in the first bullet.

Also, as I called out in the doc session -- I have yet to do a very thorough search for other people's work/terminology on this! I should probably do that before we go too deep on it!

Reply:
Okay - makes sense. I see what you're going for here and how you got there. I think you can leave it as I don't have a significantly better suggestion.